crates/apollo-schema-index/src/snapshots/apollo_schema_index__tests__search.snap

```diff
@@ -0,0 +1,10 @@
+---
+source: crates/apollo-schema-index/src/lib.rs
+expression: "results.iter().take(10).map(ToString::to_string).collect::<Vec<_>>().join(\"\\n\")"
+---
+Mutation -> Media -> MediaMetadata (6.1192875)
+Mutation -> Media -> MediaMetadata -> Dimensions (5.984665)
```

Comment:
Why does the first one receive a higher score when it doesn't contain the search term?